Eastern-Research-Group/mywaterway
# Feature/hmw 66 update services file

## Related Issues:
* https://jira.epa.gov/browse/HMW-26
* https://jira.epa.gov/browse/HMW-66
* https://jira.epa.gov/browse/HMW-75

## Main Changes:
* Updated the services.json and services-attains.json files to have the latest paths.
  * Updating the WSIO service required some code changes due to changes in attribute names.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/drinking-water
2. Verify the drinking water data loads in
3. Navigate to http://localhost:3000/national
4. Click on the "National Drinking Water" tab
5. Verify the charts load and display
6. Navigate to http://localhost:3000/community/dc/protect
7. Expand the "Watershed Health Scores" accordion
8. Verify the WSIO data displays on the map and on the right
9. Click an item on the map 
10. Verify the WSIO popup displays
11. Test the glossary
